examples/terragrunt/README.md

```diff
@@ -1,5 +1,11 @@
 # Overview
 
+## Note
+* These sample files are only deploying one `shared-services` as well as one `landing-zones`. In a real-world environment, one would most likely have more than that due to different existing environments. Nonetheless, these samples should help quickstart your Azure journey.
```

Comment:
I was thinking we could word this more like: 

'These sample files are only deploying one `shared-services` subscription as well as one landing zone. In a real-world environment, you would most likely have more than one landing zone and either one shared-services or a dev, qa, and prod shared services. Nonetheless, the samples are intended to quickstart your Azure journey. 

or we could take some parts of ^. What do you think?

Reply:
What you wrote seems very good to me, I'll make the changes!